src/store/modules/eth-base/eth-base-actions.js

```diff
@@ -83,8 +82,18 @@ export default function createActions (config) {
           tx.sign(toBuffer(context.state.privateKey))
           const serialized = '0x' + tx.serialize().toString('hex')
           const hash = api.sha3(serialized, { encoding: 'hex' })
-
-          if (!admAddress) return serialized
+          context.dispatch('createStubMessage', {
```

Comment:
This is wrong.

First, we may not have an `admAddress` (direct ETH transfer). In this case no stub is needed.

Second, stub creation and special message sending needs to be encapsulated. Create a root action, e.g. `sendSpecialMessage`, that would create a stub and send a message, and then call it from here.